tests/Feature/Api/Spells/SpellListsControllerTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use App\SpellList;
+
+class SpellListsControllerTest extends TestCase
+{
+    use WithoutMiddleware;
+    use DatabaseTransactions;
+
+    public function testUpdateSpellListNameIsAutomaticallyUpdatesSpellListName()
+    {
+        // Create the spell list
+        $payload = [
+            'name' => 'Vias de la contencion',
+            'description' => 'Debufa y paraliza a entes magicos',
+            'notes' => '- none -',
+            'list_type' => 'open'
+        ];
+        $response = $this->json('POST', '/api/spell-lists/', $payload);
+        $response->assertStatus(201);
+        $spellList = $response->decodeResponseJson()['data'];
+        // Create the spell associating the spell list
+        $payload = [
+            'name' => 'impedir miembro',
+            'level' => 3,
+            'description' => 'Paraliza un miembro del enemigo (debe haber contacto)',
+            'list_name' => $spellList['name'],
+            'code' => 'std',
+            'class' => 'F',
+            'subclass' => 'none',
+            'effect_area' => '{"code": "TARGET"}',
+            'duration' => '{"code": "TIME_LVL", "data": 3}',
+            'range' => '{"code": "CON"}',
+            'notes' => '- none -',
+            'list_id' => $spellList['id']
+        ];
+        $response = $this->json('POST', '/api/spells/', $payload);
+        $response->assertStatus(201);  
+        $spell = $response->decodeResponseJson()['data'];
+        $this->assertEquals($spellList['id'], $spell['list_id']);
+        $this->assertEquals($spellList['name'], $spell['list_name']);
+        $this->assertEquals($spell['name'], 'impedir miembro');
+        // Update Spell list name
+        $uri = ('/api/spell-lists/' . $spellList['id'] . '/');
+        # Manually testing this works. An issue with PATCH requests maybe?
```

Comment:
ah esta parte me refiero. Las trazas indican que el campo `name` es `not null` y por tanto debo pasar un valor. A no ser que ya no sepa escribir arrays asociativos en php, se lo estoy pasando. /shrug.